tests/pages/home.py

```diff
@@ -2,9 +2,54 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from selenium.webdriver.common.by import By
+
 from .base import BasePage
+from .page import PageRegion
 
 
 class HomePage(BasePage):
 
-    pass
+    _download_button_locator = (By.ID, 'download-button-desktop-release')
+
+    @property
+    def promos(self):
+        return Promos(self.selenium)
+
+    @property
+    def download_button_is_displayed(self):
+        return self.selenium.find_element(*self._download_button_locator).is_displayed()
+
+    @property
+    def upcoming_events(self):
+        return UpcomingEvents(self.selenium)
+
+
+class UpcomingEvents(PageRegion):
+
+    _root_locator = (By.ID, 'upcoming-events')
+    _next_event_locator = (By.CSS_SELECTOR, '.featured-event .event-detail > a')
+    _events_list_locator = (By.CSS_SELECTOR, '.events-list')
+
+    @property
+    def is_next_event_displayed(self):
+        return self.root.find_element(*self._next_event_locator).is_displayed()
+
+    @property
+    def is_events_list_displayed(self):
+        return self.root.find_element(*self._events_list_locator).is_displayed()
+
+
+class Promos(PageRegion):
```

Comment:
When I suggested this I was imagining a `Promo` page region that would apply to each promo. I think what you have is simple enough to not even need this page region. I think you could simply have a `total_displayed_promos` property that returns the number of promos displayed on the page. Note that this could be different from the total number in the DOM.